src/apps/irs_plan/helpers/target_areas_helper.js

```diff
@@ -24,11 +26,21 @@ const target_areas_inside_focus_filter_area = ({area_ids, selected_filter_area})
 
     if (!found_area) return false
 
-    return inside(centroid(found_area), selected_filter_area) || booleanWithin(found_area, selected_filter_area)
+    return inside(centroid(found_area), selected_filter_area) || boolean_within_multipolygons(found_area, selected_filter_area)
   })
   return result
 }
 
+function boolean_within_multipolygons(found_area, selected_filter_area) {
+  if (found_area.geometry.type === 'MultiPolygon') {
```

Comment:
What else could it be? 'Polygon'.
But anything else? There's the implicit `else`, which I guess is what we've been doing so far without problems.

Reply:
We could get ‘point’, ‘line’ or any of those as multi. But we have checks in the geodata-support to stop them from even coming through in the first place